docs/reference/commandline/swarm_update.md

```diff
@@ -21,6 +21,7 @@ Usage:  docker swarm update [OPTIONS]
 Update the swarm
 
 Options:
+      --autolock                        Enable or disable manager autolocking (requiring an unlock key to start a stopped manager)
```

Comment:
@thaJeztah: `bool` doesn't appear in the `--help` output. Should I add it anyway.